ansible/roles/build-device-worker/tasks/main.yaml

```diff
@@ -0,0 +1,28 @@
+- name: Build binary for aarch64
```

Comment:
Consider consolidating these 4 steps into one with a loop.
BTW the Makefile includes a single target `dist` that will run all 4: https://github.com/jakub-dzon/k4e-device-worker/blob/main/Makefile#L123

Reply:
That's great! I'll use that target instead. Thanks!